js_modules/dagster-ui/packages/ui-core/src/app/Util.tsx

```diff
@@ -93,10 +93,14 @@ export const formatElapsedTimeWithoutMsec = (msec: number) => {
 
 export const formatElapsedTimeWithMsec = (msec: number) => {
```

Comment:
There's a test suite at `formatElapsedTime.test.tsx`, it would be good to add some specs there.